### DISCUSSION TITLE
Master/Master Data Sharing

### DISCUSSION THREAD

**Discussion by cnighswonger** [oetiker/SmokePing#331] [Q&A] [answered]
I have a smokeping in North Carolina. A colleague has one in Granada. They each monitor a set of targets of which a subset overlap.

Is it possible to configure them to supply data to each other on the subset of targets which overlap?

**Comment by oetiker** [comment, marked as answer]
A smokeping mesh ... no, such a setup has not been envisoned ... yet

**Reply by cnighswonger** [reply]
Meshing would be very cool.

**Comment by cnighswonger** [comment]
I've added a feature request for this: #333
